examples/erc1155/tests/erc1155.rs

```diff
@@ -540,12 +641,38 @@ async fn error_invalid_array_length_in_batch_mint(
     Ok(())
 }
 
+#[e2e::test]
+async fn sets_uri(alice: Account) -> eyre::Result<()> {
```

Comment:
Will remove this test, see https://github.com/OpenZeppelin/rust-contracts-stylus/pull/416/files#r1856424663